kalaponents.variable.inc

```diff
@@ -50,6 +50,23 @@ function kalaponents_variable_info($options = array()) {
     'localize' => FALSE,
   );
 
+  // This multi-variable* holds the template paths for each component type.
+  $variable['kalaponents_templatepath_[type]'] = array(
```

Comment:
I like `template` better too, but `templatepath` is more representative of the type of information that we are storing. Sometimes the description is not enough to say that you need to add the path, also when you read that in code it's very clear that we are referring to the "template path".